certifier_service/certprotos/certifier.proto

```diff
@@ -73,6 +73,8 @@ message key_message {
   optional bytes other_key_formats          = 8;
   optional string not_before                = 9;
   optional string not_after                 = 10;
+  // Optional SEV-SNP VCEK specific fields
+  optional uint64 snp_tcb_version           = 11;
```

Comment:
Would we want a separate field for the chip id here, as well?

Reply:
Will do